app/src/main/java/com/malopieds/innertune/ui/player/Queue.kt

```diff
@@ -269,8 +267,8 @@ fun Queue(
                 WindowInsets.systemBars
                     .add(
                         WindowInsets(
-                            top = ListItemHeight,
-                            bottom = ListItemHeight,
+                            top = ListItemHeight + 12.dp + 8.dp,
+                            bottom = ListItemHeight + 8.dp,
```

Comment:
this is rather bad practice to "hardcode" values like this

Reply:
Yes, you're completely right, but I wasn't sure how it was handled in this app. In my own I always use dimension resources. But regarding the Queue maybe we could use Scaffold and put the upper and lower bars into the corresponding "topBar" and "bottomBar" letting the Scaffold measure their heights, so theres no need for a custom layout?

Reply:
But regarding the 8.dp spacing ... I mean its hardcoded all over the app:

![Screenshot 2024-07-25 08:28:38](https://github.com/user-attachments/assets/d611019b-519b-4e75-b43a-c0bad52f0a3e)

I personally favor dimension resources, but it doesn't seem to be used here and what would be the alternative?